react/components/SliderTrack.tsx

```diff
@@ -1,10 +1,16 @@
 import React, { FC, useEffect, useRef } from 'react'
 import { useListContext } from 'vtex.list-context'
-import { useCssHandles } from 'vtex.css-handles'
+import { useCssHandles, applyModifiers } from 'vtex.css-handles'
 
 import { useSliderState } from './SliderContext'
 
-const CSS_HANDLES = ['sliderTrack', 'slide', 'slideChildrenContainer']
+const CSS_HANDLES = [
+  'sliderTrack',
+  'slide',
+  'slideChildrenContainer',
+  'firstVisible',
+  'lastVisible',
```

Comment:
No need for these two since they are not new handles, but modifiers on the `slide` handler